__init__.py

```diff
@@ -56,6 +55,13 @@ def __init__(self):
         self.volume_sound = join(dirname(__file__), "blop-mark-diangelo.wav")
         self.vol_before_mute = None
         self._mixer = None
+        volume_words = self.translate_namedvalues('volume.words', delim=',')
+        if volume_words != {}:
+            new_volume_words = {}
+            for volume_words_key in self.VOLUME_WORDS:
+                if volume_words.get(volume_words_key) != None:
+                    new_volume_words[volume_words.get(volume_words_key)] = self.VOLUME_WORDS[volume_words_key]
+            self.VOLUME_WORDS = new_volume_words
```

Comment:
It feels like this might be better contained in its own function to aid in readability and show that it's self-contained. 
Perhaps something like `self.translate_volume_words()`?